packages/synapse-react-client/src/style/components/_synapse-homepage.scss

```diff
@@ -70,11 +77,16 @@
       padding: 30px 40px;
     }
 
+    .MuiButton-root {
+      padding-top: 0.75rem;
+      padding-bottom: 0.75rem;
+    }
```

Comment:
Maintain button padding previously provided by bootstrap-4-backport when viewport is small